src/kili/log/logging.py

```diff
@@ -0,0 +1,7 @@
+import logging
+import sys
+
+logger = logging.getLogger("kili")
```

Comment:
you can use this logger everywhere in the codebase

replace the several `getLogger( kili something, or __name__ )`  with the new global logger

https://github.com/search?q=repo%3Akili-technology%2Fkili-python-sdk%20getLogger&type=code

Reply:
yes i'll done it in a second MR